NewSpring/ApollosProject
# Updates project to use Apollos Core custom icon support

## DESCRIPTION
This PR deprecates `NSIcon` in favor of the native Apollos Core custom icon support. Please review any additional code you may have copied over from Core that may no longer be needed for custom icon support. As far as I know, all unneeded code has been removed in this PR.
 
### What does this PR do, or why is it needed?
- Replaces all references to `NSIcon` with `Icon`
- Deletes any unnecessary code used to enabled use of `NSIcon`
- Renames references to `leaf-logo` to the Apollos Core standard naming of `brand-icon`. This simplifies and streamlines integration with the rest of Apollos Core.
### How do I test this PR?

Open app  and verify that all files changed are rendered as expected.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
